ts/image-occlusion/review.ts

```diff
@@ -111,6 +111,12 @@ function setupImageOcclusionInner(setupOptions?: SetupImageOcclusionOptions): vo
         } else {
             button.style.display = "none";
         }
+
+        window.addEventListener("keydown", (event) => {
+            if (event.key === "h") {
+                toggleMasks();
```

Comment:
Some users use hjkl for reviewing, which this will break. Maybe we could use a capital M instead?

Reply:
It can be done.